docs/jetstream/configuration.md

```diff
@@ -203,6 +203,11 @@ description = "Number of cows clicked"
 # user experience as a result of the experiment; it is a subset of enrollments.
 # We may default to an exposures basis in the future.
 exposure_basis = ["exposures", "enrollments"]
+
+# Metrics can depend on other metrics that need to be referenced.
+# When a metric depends on upstream metrics, select_expression and
+# data_source are optional. Usually, upstream metrics get used when computing statistics.
```

Comment:
What happens in the case when `depends_on` is provided alongside `select_expression` and `data_source`?

Reply:
Should this go in the docs?